pkg/apis/kops/validation/legacy.go

```diff
@@ -436,6 +436,17 @@ func ValidateCluster(c *kops.Cluster, strict bool) *field.Error {
 		//      return field.Required(kubeProxyPath.Child("Master"), "")
 		//}
 
+		// @check if are using kubernetes <1.9.3 with ipvs and deny canal 3.2.3 due to issue
+		// https://docs.projectcalico.org/v3.2/getting-started/kubernetes/requirements
+		if c.Spec.KubeProxy.ProxyMode == "ipvs" {
+			if kubernetesRelease.LT(semver.MustParse("1.9.3")) {
+				// @NOTE: i'm not sure about this one, it could block someone from upgrading i.e. running 1.9.2 + canal + with pre GA ipvs .. perhaps drop?
```

Comment:
It's probably better to update the `bootstrapchannelbuilder` for Canal v3 to only apply on K8s v1.9.3+ and leave the v2.6 deployment to k8s versions below that (`KubernetesVersion: ">=1.9.3"`) .

Reply:
Yep... that must so much more sense!